stglib/core/utils.py

```diff
@@ -1301,14 +1301,27 @@ def spcon_from_salinity(S):
     )
 
 
-def check_fits_in_int32(ds, var):
-    if np.nanmax(np.abs(ds[var])) > (2**31 - 1):
-        warnings.warn(
-            f"32-bit integer overflow on {var}; setting encoding to i4 will fail"
-        )
-        return False
+def check_fits_in_int32(ds, var, units="s"):
```

Comment:
Whoops, sorry, I could have been more clear. There is already a check_time_fits_in_int32 function, defined below. Please use that one for time rather than changing this func def (unless there is something else I am missing).